modules/aggregation-layers/src/grid-layer/grid-cell-layer-vertex.glsl.ts

```diff
@@ -0,0 +1,68 @@
+export default `#version 300 es
```

Comment:
If you haven't started using the syntax highlight plugins (assuming you are using vscode) I recommend it.

```suggestion
export default /* glsl */ `\
#version 300 es
```

See luma.gl docs: https://luma.gl/docs/developer-guide/editing#shader-syntax-highlighting